pkg/mcs/metastorage/server/grpc_service.go

```diff
@@ -110,6 +110,7 @@ func (s *Service) Watch(req *meta_storagepb.WatchRequest, server meta_storagepb.
 			return nil
 		case res := <-watchChan:
 			if res.Err() != nil {
+				log.Error("watch channel failed", zap.Error(res.Err()))
```

Comment:
Is it necessary to use the error level log?  If yes,  you should explain how to solve it?